crates/swc_ecma_minifier/src/compress/optimize/inline.rs

```diff
@@ -651,31 +652,8 @@ where
                         None
                     }
                 })
-                .and_then(|v| {
-                    // Prevent infinite recursion.
-                    let ids = idents_used_by(&**v);
-                    if ids.contains(&i.to_id()) {
-                        None
-                    } else {
-                        Some(v)
-                    }
-                })
                 .cloned()
             {
-                match &*value {
-                    Expr::Lit(Lit::Num(..)) => {
-                        if self.ctx.is_lhs_of_assign {
-                            return;
-                        }
-                    }
-                    Expr::Member(..) => {
-                        if self.ctx.executed_multiple_time {
```

Comment:
I remember the reason I added this code, but not sure because situation is changed quite a lot

Reply:
We don't inline member expression at all.